Makefile

```diff
@@ -21,10 +21,15 @@ test: install
 test-race: | test
 	go test -race -mod=readonly ./...
 
+test-repeated: | test
```

Comment:
please remove this, if the test is flakey we'll know about it over multiple Ci runs (ask me how I know :)